include/net/nrf_cloud_pgps.h

```diff
@@ -254,7 +254,7 @@ int nrf_cloud_pgps_process(const char *buf, size_t buf_len);
  * @return 0 if successful, otherwise a (negative) error code.
  */
 int nrf_cloud_pgps_inject(struct nrf_cloud_pgps_prediction *p,
-			  const struct gps_agps_request *request);
+			  const struct nrf_modem_gnss_agps_data_frame *request);
 
```

Comment:
Same as above, please update `nrf\applications\serial_lte_modem\src\gnss\slm_at_gnss.c`

Reply:
Already included in this PR.